### ISSUE TITLE
display pagination for app/ and db/ operations lists

### ISSUE THREAD

**Issue by bantic** [aptible/diesel.aptible.com#49]
The API will return pagination info for collection resources (see https://github.com/aptible/api.aptible.com/pull/146). Diesel should use these to display pagination links, starting with the `apps/:id/operations and` and `databases/:id/operations` history pages.

**Comment by bantic** [comment]
I'll assume the API will respond to a `GET /apps/:id/operations` request with a payload like:
```javascript
{
  page: 1,
  total_pages: 10,
  per_page: 10,
  _embedded: {
  }
}
```

**Comment by bantic** [comment]
@sandersonet Is the "app activity" (and db activity) page the place where the paginated list of operations will end up in you 2.0 designs, i.e.:

![dashboard-2 0 0 1 sketch 2015-01-14 13-21-48](https://cloud.githubusercontent.com/assets/2023/5744739/56ec5f80-9bf1-11e4-8642-c1cc0d190822.jpg)

Do you have a UX or UI in mind for the pagination? This PR adds very simple pagination (prev current-page next) as shown in the image ^^

**Comment by sandersonet** [comment]
@bantic Correct, 'activity' is the new page name for app/db operations.   

For now, just use the bootstrap default: http://getbootstrap.com/components/#pagination for pagination links.  Someday we can consider an infinite scroll UX.

**Comment by bantic** [comment]
PR over at #60 .

Will do the basic bootstrap styling separately.

**Comment by bantic** [comment]
closed in #60
